### PR TITLE
Expand selenium OpenAPI UI tests

### LINKED ISSUE #27052: Expand mpOpenAPI Selenium tests to cover more of the manual test case scenarios
Currently we have a document that outlines all the manual steps for validating updates to the the MicropProfile OpenAPI UI - https://github.com/OpenLiberty/openapi-ui-test-app#for-the-microprofile-ui.

While it does not take that long to go through, it is quite tedious and has the potential to be error prone, especially given the nature of checking the CSS on certain elements for their colour. 

We should attempt to increase the selenium coverage of a number of these aspects to further reduce the burden upon the process of doing Swagger-UI updates.

Both com.ibm.ws.openapi.ui_fat and io.openliberty.microprofile.openapi.ui.internal_fat should be tested given the manual process covers these as well. 

The test app in both of these is overly simple in that it has a single operation, we would need to expand to at least cover PUT, POST and DELETE.

Currently this task is to improve our automated coverage of the UI, so we can focus manual testing on elements that will be harder to test such as Filtering.

### PR DESCRIPTION
**Pull request by ryan-storey**
Expanding our OpenAPI UI selenium tests for mpOpenAPI and public and private versions to reduce manual testing
Closes #27052

### CODE REVIEW

**Comment by ryan-storey** [comment]
#libby #build

**Comment by ryan-storey** [comment]
#libby #build

File: dev/com.ibm.ws.openapi.ui_fat/fat/src/com/ibm/ws/openapi/ui/fat/tests/UIBasicTest.java [hunk 1]
```diff
@@ -123,6 +124,7 @@ private void testUI() {
         WebElement headerbar = waitForElement(driver, By.cssSelector("div.headerbar"));
         assertEquals("Headerbar colour", Color.fromString("#191c2c"), Color.fromString(headerbar.getCssValue("background-color")));
 
+        System.out.println(headerbar.getAttribute("innerHTML"));
```
Comment (abutch3r) [inline dev/com.ibm.ws.openapi.ui_fat/fat/src/com/ibm/ws/openapi/ui/fat/tests/UIBasicTest.java]: remove debug statement

Comment (ryan-storey) [reply dev/com.ibm.ws.openapi.ui_fat/fat/src/com/ibm/ws/openapi/ui/fat/tests/UIBasicTest.java]: Completely missed that, thank you

**Comment by ryan-storey** [comment]
#libby #build

**Comment by ryan-storey** [comment]
#libby #build

**Comment by ryan-storey** [comment]
#libby #build

**Comment by ryan-storey** [comment]
#pbbeta

**Review by abutch3r — CHANGES_REQUESTED** [review]
The three bundle files need to be removed before this can be merged - i suspect there is a commit that hasn't been applied properly that would of removed these three files. 

The rest of the changes look good and ready to go.

**Comment by ryan-storey** [comment]
#libby #build

**Comment by ryan-storey** [comment]
#pbbeta

**Review by abutch3r — APPROVED** [review]
LGTM

**Pull request merged**